docker/Dockerfile.testing

```diff
@@ -9,12 +9,12 @@ USER root
 
 # Use 'update' for existing and 'install' for new package.
 # Keep same channel ordering to not revert anything.
-RUN umask 0000 && \
-    conda config --prepend channels nodefaults || true && \
-    conda config --remove channels defaults || true && \
-    pip uninstall -y hsclient && \
-    mamba install --force-reinstall -c conda-forge -c cdat -c bokeh -c fortiers ravenpy==0.18.1 sparse==0.15.5 && \
-    mamba clean --all --yes
+# RUN umask 0000 && \
```

Comment:
Not sure why you want to comment out this because next time you use it, will you commit again to disable after use?  I am fine with both.

Reply:
I think it's just to show that these are examples of what _can_ be done. With the latest changes, these uninstall/reinstall steps are obsolete.

Reply:
They are always out-of-date with latest.  They are only applicable against the base image specified in `FROM`.  Each time we use this, we  have to always update the `FROM`.

This was just a remark for later.  I am not asking to revert this change.